src/match/queue.ts

```diff
@@ -0,0 +1,48 @@
+import { randomUUID } from 'crypto';
+
+export const subscribe = () => {};
+type Item = {
+  id: string;
+  joinAt: number;
+};
+
+export class Queue {
```

Comment:
Let's call it `MatchMakingQueue` or something more descriptive. It isn't just any generic Queue that we can use for normal data structure stuff!